tools/create-extension/_package.json

```diff
@@ -20,11 +20,11 @@
 		"test:unit": "wp-scripts test-unit-js"
 	},
 	"devDependencies": {
-		"@wordpress/scripts": "^12.2.1",
-		"@woocommerce/eslint-plugin": "1.1.0",
-		"@woocommerce/dependency-extraction-webpack-plugin": "1.6.0"
+		"@wordpress/scripts": "^25.1.0",
+		"@woocommerce/eslint-plugin": "2.2.0",
+		"@woocommerce/dependency-extraction-webpack-plugin": "2.2.0"
 	},
 	"dependencies": {
-		"@wordpress/hooks": "^3.5.0"
+		"@wordpress/hooks": "^3.24.0"
```

Comment:
Moving this tool to `@woocommerce/create-woo-extension` solves the problem of having to keep up dependencies like this.